basilisk/gui/conversation_tab.py

```
@@ -1158,6 +1171,261 @@
 				invalid_found = True
 		return not invalid_found
 
+	def create_ocr_widget(self):
+		"""Create and configure the OCR button"""
```

Ruff (0.8.2)

First line should end with a period, question mark, or exclamation point

Add closing punctuation

(D415)